deployment-local/local-tg

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+if [ "$1" = "startup-jenkins" ]; then
+    echo "__          _______  ____ ___    _______        _    _____      _     _    _       ____   _____   ___   _      "
+    echo "\ \        / / ____|/ __ \__ \  |__   __|      | |  / ____|    (_)   | |  | |    /  __  \/ _____|/    \| |     "
+    echo " \ \  /\  / / (___ | |  | | ) |    | | ___  ___| |_| |  __ _ __ _  __| |  | |    | |  | | |     | /__\ | |     "
+    echo "  \ \/  \/ / \___ \| |  | |/ /     | |/ _ \/ __| __| | |_ |  __| |/ _  |  | |    | |  | | |     | ____ | |     "
+    echo "   \  /\  /  ____) | |__| / /_     | |  __/\__ \ |_| |__| | |  | | (_| |  | |____| |__| | |____ | |  | | |____ "
+    echo "    \/  \/  |_____/ \____/____|    |_|\___||___/\__|\_____|_|  |_|\____|  |______|\ ___ /\______|_|  |_|______|"
+    echo ""
+
+    chmod +x ./tomcat/apache-tomcat-8.5.43/bin/*
+    chmod +x ./install_jenkins_plugins.sh
+    chmod +x ./add_credentials.sh
+    chmod +x ./add_config_provider_files.sh
+    chmod +x ./testgrid/testgrid-home/testgrid-dist/WSO2-TestGrid/*
+
+    if [ $(curl -s -o /dev/null -w "%{http_code}" http://localhost:8080) -eq 000 ]
+    then
+
+      ./tomcat/apache-tomcat-8.5.43/bin/startup.sh
+
+    	while [ $(curl -s -w "%{http_code}" http://localhost:8080/admin -o /dev/null) -eq 000 ]
+    	do
+      	  sleep 1
+    	done
+
+    	echo "installing plugins....."
+    	./install_jenkins_plugins.sh
+
+    	echo "adding credentials....."
+    	./add_credentials.sh
+
+    	echo "adding config provider files...."
+    	./add_config_provider_files.sh
+
+    else
+      echo "already running..."
+    fi
+
+
+    if [ $(curl -s -o /dev/null -w "%{http_code}" http://localhost:8082) -eq 000 ]
+    then
+        echo "starting H2 server....."
+        java -jar lib/h2-*.jar  -baseDir ./h2dbs &> /dev/null &
+    fi
+
+
+elif [ "$1" = "shutdown-jenkins" ]
+then
+
+   if [ $(curl -s -o /dev/null -w "%{http_code}" http://localhost:8080) -eq 302 ]
+   then
+     ./tomcat/apache-tomcat-8.5.43/bin/shutdown.sh
+   fi
+
+   if [ $(curl -s -o /dev/null -w "%{http_code}" http://localhost:8082) -eq 200 ]
+   then
+     java -cp lib/h2-*.jar org.h2.tools.Server -tcpShutdown tcp://localhost:9092
+   fi
+
+else
+   echo "Invalid command"
```

Comment:
There's an issue here in terms of user experience.

As a first time user, I ran the ./local-tg command. It has simply output "Invalid command", and exited. I was left wondering what to do next.
![image](https://user-images.githubusercontent.com/936037/65874581-9c1f9280-e3a3-11e9-9055-88888ef1e192.png)

Please help text such that user will know what to do next. To know how a command-line should behave, analyze the log output of the wso2 'wum' client. Here's a sample screenshot for reference.

![image](https://user-images.githubusercontent.com/936037/65874812-f15ba400-e3a3-11e9-87d6-2072b4fc133d.png)